lib/postmark/http_client.rb

```diff
@@ -19,7 +19,7 @@ class HttpClient
       :path_prefix => '/',
       :http_read_timeout => 15,
       :http_open_timeout => 5,
-      :http_ssl_version => :TLSv1
+      :http_ssl_version => (RUBY_VERSION >= '2.0.0' ? :TLSv1_2 : :TLSv1)
```

Comment:
I noticed mentions of max_version, min_version options. I see these in Ruby 2.4.0 and above.
Wondering if we should set version in these higher versions this way. 

https://github.com/wildbit/postmark-gem/issues/86